<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsClassMySelectGenshinImpactHandle</class>
 <widget class="QWidget" name="QtWidgetsClassMySelectGenshinImpactHandle">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择原神窗口</string>
  </property>
  <property name="windowIcon">
   <iconset resource="GenshinImpact_AutoMap.qrc">
    <normaloff>:/IconExit/resource/icon16.ico</normaloff>:/IconExit/resource/icon16.ico</iconset>
  </property>
  <widget class="QPushButton" name="ButtonOk">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>138</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCancel">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>138</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHandle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>窗口句柄</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>5</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSelect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>选择
原神</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>窗口标题</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelClass">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>窗口类名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditClass">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>65</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHandle">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>35</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GenshinImpact_AutoMap.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsClassMySelectGenshinImpactHandle</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsClassMySelectGenshinImpactHandle</receiver>
   <slot>SelectGenshinImpactWndOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonSelect</sender>
   <signal>pressed()</signal>
   <receiver>QtWidgetsClassMySelectGenshinImpactHandle</receiver>
   <slot>SelectGenshinImpactWndStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SelectGenshinImpactWndOk()</slot>
  <slot>SelectGenshinImpactWndStart()</slot>
 </slots>
</ui>
